bentoml/_internal/cli/model_management.py

```diff
@@ -170,29 +170,34 @@ def delete_target(target):
     @model_cli.command(help="Export Model to a tar file")
```

Comment:
could you add an sample CLI command in the help message? it would be great to show user an example command in the help message of `bentoml export --help`